### PR TITLE
Define structure Vss for setup tool

### PR DESCRIPTION
**Pull request by Yamaguchi**

### CODE REVIEW

File: src/crypto/vss.rs [hunk 1]
```diff
@@ -0,0 +1,270 @@
...
+                positive_commitments: positive_commitments,
+                positive_secret: positive_secret,
+                negative_commitments: negative_commitments,
+                negative_secret: negative_secret,
+            }
+        }
+}
+
+#[derive(Clone)]
+pub struct Commitment {
```
Comment (rantan) [inline src/crypto/vss.rs]: Is there any reason to create this struct, not to use GE directly? 

Creating this struct is no problem. It's just my curiosity.

File: src/crypto/vss.rs [hunk 2]
```diff
@@ -0,0 +1,270 @@
...
+            positive_commitments, 
+            positive_secret, 
+            negative_commitments, 
+            negative_secret,
+        ))
+    }
+}
+
+impl Encodable for Commitment {
+    fn consensus_encode<S: io::Write>(&self, mut s: S) -> Result<usize, encode::Error> {
```
Comment (rantan) [inline src/crypto/vss.rs]: How about using 33bytes compressed public key format. It decrease data size 31bytes.

File: src/crypto/vss.rs (hunk 1 above)
Comment (Yamaguchi) [reply src/crypto/vss.rs]: To define method for encoding/decoding commitment.
We cant implement the method consensus_decode/consensus_encode on GE directly, right?

Comment (rantan) [reply src/crypto/vss.rs]: Yes, it is right. But encoding/decoding for a commitment can be included in VSS encoding/decoding function.

File: src/crypto/vss.rs (hunk 2 above)
Comment (Yamaguchi) [reply src/crypto/vss.rs]: For now, Commitment is converted to GE and GE is used internally.
I think it is natural to use 64 bytes format for encoding/decoding GE.

Comment (rantan) [reply src/crypto/vss.rs]: I see. keep as it is.

Using 33bytes format has a pro but it is not big problem for now.

**Pull request merged**